gatsby-ssr.js

```diff
@@ -42,6 +42,12 @@ export const onRenderBody = ({ setHeadComponents }) => {
         enablePartytown
       />,
       <Partytown key="party" />,
+      <link
+        key="preconnect"
+        rel="preconnect"
+        crossOrigin
+        href="https://googletagmanager.com"
+      />,
```

Comment:
I think this should have its own PR. It's not related to the Filter component.

Reply:
We had https://github.com/vtex-sites/base.store/pull/233 but we merged this PR before.